### PR TITLE
Liquibase Actions

### PR DESCRIPTION
**Pull request by alismx**
# DEVOPS PULL REQUEST

## Related Issue

- Improve our ability to manage our database, monitor the current state, and respond to incidents

## Changes Proposed

- Updates our database rollback workflow to allow us to validate and clear checksums when needed without needing to start up a VM.
- Updates our tests for the DB changelog to run specific tasks in a similar way they would be run against a live database.
- Update the naming scheme for our database actions.
- add an image_action variable to our terraform, if we need more actions in the future, we'd only need to update the `db_liquibase_action.sh` script.

## Additional Information

- At some point in the near future, I'd like to rename the jobs to represent better what they're doing.
- At some point soon, I'd like to set up a `validation` run against an environment after it's been deployed.
- These two things are written out in [this ticket](https://app.zenhub.com/workspaces/devsecops-63d830363b26890016306a3f/issues/gh/cdcgov/prime-simplereport/5666)

## Testing

- feel free to run the `Rollback DB` job on this branch against your favorite dev environment. If the options aren't clear, please reach out so I can understand what changes I can make to improve it!

### Infrastructure
- [ ] Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!

### Security
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

### Cloud
- [ ] Oncall has been notified if this change is going in after-hours
- [ ] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### Documentation
- [ ] Any changes to the startup configuration have been documented in the README

### CODE REVIEW

**Comment by alismx** [comment]
I've found an issue with this PR. For now, don't review, but I'll get a fix pushed soon!

**Comment by alismx** [comment]
This has been validated, able to `validate`, `clear-checksums` and `rollback`. Please feel free to review!

File: .github/workflows/rollbackDB.yml [hunk 1]
```diff
@@ -3,24 +3,35 @@ name: Rollback DB
...
+        default: dev
+      action:
+        description: |
+          The liquibase action to run.
+        required: true
+        type: choice
+        options: [validate, clear-checksums, rollback]
+        default: validate
+      tag:
+        description: |
```
Comment (emyl3) [inline .github/workflows/rollbackDB.yml:21]: I'm wondering if there's a way to conditionally require this field for when a user selects the rollback option? 🤔

Comment (emyl3) [reply .github/workflows/rollbackDB.yml:21]: i ran it without a tag to see what would happen https://github.com/CDCgov/prime-simplereport/actions/runs/4893957156/jobs/8737569806

**Review by emyl3 — APPROVED** [review]
LGTM! Tried all three commands on dev7! :D This is awesome! Thank you alis 😹 

Left one non-blocking comment 😃

File: .github/workflows/rollbackDB.yml (hunk 1 above)
Comment (alismx) [reply .github/workflows/rollbackDB.yml:21]: Good thinking, I think the best we could do would be to check in bash before we deploy the terraform. I think I'll leave that for a future iteration for now.

**Review by emyl3 — APPROVED** [review]
🎉 adding approval back :)

**Review by BobanL — APPROVED** [review]
I've also tested these changes and it all seems to work as expected! Thanks for your work on this 😃

**Pull request merged**